ads/google/a4a/traffic-experiments.js

```diff
@@ -81,35 +92,59 @@ export function googleAdsIsA4AEnabled(win, element, experimentId, branches) {
 }
 
 /**
- * Set experiment state from URL parameter, if present.
+ * Set experiment state from URL parameter, if present.  This looks for the
+ * presence of a URL parameter of the form
+ *   `exp=expt0:val0,expt1:val1,...,a4a:X,...,exptN:valN`
+ * and interprets the X as one of the following:
+ *   - `-1`: Manually-triggered experiment.  For testing only.  Sets
+ *     `adtest=on` on the ad request, so that it will not bill or record
+ *     user clicks as ad CTR.  Ad request will be accounted in a special
+ *     'testing only' experiment statistic pool so that we can track usage
+ *     of this feature.
+ *   - `0`: Ad is explicitly opted out of the overall A4A-vs-3p iframe
+ *     experiment.  Ad will serve into a 3p iframe, as traditional, but ad
+ *     request and clicks will not be accounted in experiment statistics.
+ *   - `1`: Ad is on the control branch of the overall A4A-vs-3p iframe
+ *     experiment.  Ad will serve into a 3p iframe, and ad requests and
+ *     clicks _will_ be accounted in experiment statistics.
+ *   - `2`: Ad is on the experimental branch of the overall A4A-vs-3p iframe
+ *     experiment.  Ad will render via the A4A path, including early ad
+ *     request and (possibly) early rendering in shadow DOM or iframe.
  *
- * @param {!Window} win
- * @param {string} experimentId
- * @param {!Branches} branches
- * @visibleForTesting
+ * @param {!Window} win  Window.
+ * @param {!string} experimentName  Name of the overall experiment.
```

Comment:
They're actually semantically different, unfortunately.  `experimentName` is a human-readable string like `expAdsenseA4A` (see `adsense-a4a-config.js#ADSENSE_A4A_EXPERIMENT_NAME`).  It's used to group together all related branches and to set them in the `experiments.js` API.

The `experimentId`s are machine-readable numeric IDs like `MANUAL_EXPERIMENT_ID`.